src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure implements GetArea {
```

Comment:
```suggestion
public class RightTriangle extends Figure  {
```
аналогічно у інших класах